snippets/puppet_setup.erb

```diff
@@ -32,7 +34,15 @@ cat > <%= etc_path %>/puppet.conf << EOF
 <%= snippet 'puppet.conf' %>
 EOF
 
-<% unless @host.param_true?('enable-puppetlabs-pc1-repo') -%>
+<% if @host.param_true?('enable-puppetlabs-pc1-repo') -%>
+<% if rhel_compatible -%>
```

Comment:
Given these calls are idempotent I think this is safe for Fedora and we can remove this distinction.